homeassistant/components/sonos/services.yaml

```diff
@@ -124,59 +131,82 @@ set_option:
 play_queue:
   name: Play queue
   description: Start playing the queue from the first item.
+  target:
+    entity:
+      integration: sonos
+      domain: media_player
   fields:
-    entity_id:
-      description: Name(s) of entities that will start playing.
-      example: "media_player.living_room_sonos"
-      selector:
-        entity:
-          integration: sonos
-          domain: media_player
     queue_position:
+      name: Queue position
       description: Position of the song in the queue to start playing from.
       example: "0"
       selector:
         number:
           min: 0
-          max: 100000000
+          max: 100
           mode: box
 
 remove_from_queue:
   name: Remove from queue
   description: Removes an item from the queue.
+  target:
+    entity:
+      integration: sonos
+      domain: media_player
   fields:
-    entity_id:
-      description: Name(s) of entities that will remove an item.
-      example: "media_player.living_room_sonos"
-      selector:
-        entity:
-          integration: sonos
-          domain: media_player
     queue_position:
+      name: Queue position
       description: Position in the queue to remove.
       example: "0"
       selector:
         number:
           min: 0
-          max: 100000000
+          max: 100
```

Comment:
Why change the limit of the position in the queue?

Reply:
@frenck Going back to what you said before about remote services, does a 100 million steps matter if we are using a box instead of a slider?

Reply:
100 might be excessive but 100 feels a bit low IMO. 1000 or 10000 seems more reasonable to me.